src/common/execute.ts

```diff
@@ -24,7 +24,7 @@ export const executeAsync = async (
       await context.request(
         GraphqlActions.asyncSessionStatus,
         { sessionId },
-        { customEnvironment: DEFAULT_ENVIRONMENT_NAME },
+        { customEnvironment: RequestHeaderIgnored, customWorkspaceId: RequestHeaderIgnored },
```

Comment:
migration commit by token auth works if send here the options param

it is works if `customEnvironment` not matter: 
`result = (await context.request(GraphqlActions.asyncSessionStatus, { sessionId }, options)).status`;

Reply:
I suppose, because we use token token. We have to ignore it in the request, too

Reply:
I have some problem still

![image](https://user-images.githubusercontent.com/77967415/122950389-14529680-d385-11eb-9b20-0ee47a9d2c3d.png)